package.json

```diff
@@ -184,6 +225,20 @@
 					],
 					"default": "beside",
 					"description": "Where to place the native symbol windows"
+				},
+                "godot_tools.scenePreview.previewRelatedScenes": {
```

Comment:
Mixed indentation – please use tabs in the whole file.

Reply:
I want it on the record that I did spend several hours trying to force Prettier to format to our style, and all I accomplished was getting it to stop working entirely.